### PR TITLE
I#3018

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #3018 
Reopened case #2898 

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/high-availability.md [hunk 1]
```diff
@@ -64,9 +67,13 @@ For a Front End pool to function at all, it cannot be in pool-level quorum loss.
 |7  <br/> |Any 4  <br/> |
 |8-9  <br/> |Any 4 of the first 7 servers  <br/> |
 |10-12  <br/> |Any 5 of the first 9 servers  <br/> |
+|12-16  **For Skype For business 2019**  <br/> |Any 7 of the first 12 servers  <br/> |
    
 In the preceding table, the "first servers" are the servers which were brought up first, chronologically, when the pool was started for the first time. To determine these servers, you can use the  `Get-CsComputer` cmdlet with the `-PoolFqdn` option. This cmdlet will show the servers in the order that they appear in the topology, and the ones at the top of the list are the first servers.
   
+> [!IMPORTANT]
+> It's always relevant to know that the maximum front end number of servers has been increased to 16 on the [new skype for business 2019](https://docs.microsoft.com/skypeforbusiness/plan/user-model-2019)
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/high-availability.md]:
```suggestion
> The maximum number of front end servers has been increased to 16 in [Skype for Business Server 2019](https://docs.microsoft.com/skypeforbusiness/plan/user-model-2019)
```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/high-availability.md [hunk 2]
```diff
@@ -46,6 +46,9 @@ The first time you start a new Front End pool, it is essential that 85% of the s
 |10  <br/> |8  <br/> |
 |11  <br/> |9  <br/> |
 |12  <br/> |10  <br/> |
+|16 **For Skype For business 2019** <br/> |12  <br/> |
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/high-availability.md]:
```suggestion
|16 **For Skype for Business Server 2019** <br/> |12  <br/> |
```

File: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/high-availability.md [hunk 3]
```diff
@@ -64,9 +67,13 @@ For a Front End pool to function at all, it cannot be in pool-level quorum loss.
 |7  <br/> |Any 4  <br/> |
 |8-9  <br/> |Any 4 of the first 7 servers  <br/> |
 |10-12  <br/> |Any 5 of the first 9 servers  <br/> |
+|12-16  **For Skype For business 2019**  <br/> |Any 7 of the first 12 servers  <br/> |
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/high-availability.md]:
```suggestion
|12-16  **For Skype for Business Server 2019**  <br/> |Any 7 of the first 12 servers  <br/> |
```

**Comment by jambirk** [comment]
@j0rt3g4  this looks like a reasonable fix for the issue,

**Pull request merged**